app/components/application_component.rb

```diff
@@ -0,0 +1,2 @@
+class ApplicationComponent < ViewComponent::Base
```

Comment:
Seems like a good idea. Should we modify the other views that already inherit from `ViewComponent::Base`?